spacemacs/packages.el

```diff
@@ -2492,7 +2492,7 @@ displayed in the mode-line.")
 
 (defun spacemacs/init-volatile-highlights ()
   (use-package volatile-highlights
-    :init
+    :config
```

Comment:
I have an intuition that use-package wraps the config section in an eval-after-load, which ensures that the package will be installed first. Just reading the source now to see whether that's true.